.idea/huggingface_sb3.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think things under `.idea` come from IntelliJ. Same for `inspectionProfiles`. I would remove those dirs

Reply:
Done :heavy_check_mark: